### PR TITLE
GitHub actions 2

### PR DESCRIPTION
**Pull request by amineoualialami**

### CODE REVIEW

File: posts/deployToEcsWithGithubAction.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can become a challenge for architecture and plateform infrastructure teams, due to the integration complexity with many platforms: Cource Control, CI/CD, Artifact Repositories, Quality, SAST/SCA, Deployment, etc. Many DevOps leader believes that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.
```
Comment (yhippa) [inline posts/deployToEcsWithGithubAction.md]:
```suggestion
Building a DevOps platform nowadays can become a challenge for architecture and platform infrastructure teams, due to the integration complexity with many platforms: Cource Control, CI/CD, Artifact Repositories, Quality, SAST/SCA, Deployment, etc. Many DevOps leader believes that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.
```

File: posts/deployToEcsWithGithubAction.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+That's why tech giants are trying to integrate all pipeline steps into their platforms.
+In this article, we will deploy a simple [Spring Boot](https://spring.io/projects/spring-boot) application to [Amazon Elastic Container Service (ECS)](https://spring.io/projects/spring-boot) using the new GitHub feature called [GitHub Actions](https://github.com/features/actions).
+
+# What is GitHub Actions?
+GitHub Actions enables you to create custom software development lifecycle pipelines directly in your GitHub repository.
+You can write individual tasks, called "actions", and combine them to create a custom pipeline.
```
Comment (yhippa) [inline posts/deployToEcsWithGithubAction.md]:
```suggestion
You can write individual tasks, called "actions" and combine them to create a custom pipeline.
```

File: posts/deployToEcsWithGithubAction.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+Behind the scenes, your pipeline is run on GitHub-hosted machines, called "runners".
+
+
+# Create Your First Pipeline With Github Actions
+On your GitHub repository select the **Actions** tab.
+GitHub propose popular pipelines to start with. We will choose: **Deploy to Amazon ECS**
```
Comment (yhippa) [inline posts/deployToEcsWithGithubAction.md]:
```suggestion
GitHub proposes popular pipelines to start with. We will choose: **Deploy to Amazon ECS**
```

File: posts/deployToEcsWithGithubAction.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+      uses: aws-actions/amazon-ecs-deploy-task-definition@v1
+      with:
+        task-definition: ${{ steps.task-def.outputs.task-definition }}
+        service: dc-dojo-service-github
+        cluster: dcdojo-cluster
+        wait-for-service-stability: true
+
+```
+
+This pipeline  will checkout the code, build the JAR file, build and push the Docker image to [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/), and then  deploy a new task definition to Amazon ECS, on every push to the master branch.
```
Comment (yhippa) [inline posts/deployToEcsWithGithubAction.md]:
```suggestion
This pipeline will checkout the code, build the JAR file, build and push the Docker image to [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/), and then deploy a new task definition to Amazon ECS on every push to the master branch.
```

File: posts/deployToEcsWithGithubAction.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can become a challenge for architecture and platform infrastructure teams, due to the integration complexity with many platforms: Cource Control, CI/CD, Artifact Repositories, Quality, SAST/SCA, Deployment, etc. Many DevOps leader believes that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.
```
Comment (yhippa) [inline posts/deployToEcsWithGithubAction.md]: I was going to make a change to this but it feels like my changes from two days ago never got made and there are new issues with this now.  Is this a new repo? I'd rather not make the same exact comments from the first time.

Comment (amineoualialami) [reply posts/deployToEcsWithGithubAction.md]: Yes, it was another pull request, I found your last comments about this part and did the asked changes.
Please let me know if you see anything else.
Thanks

File: posts/deployToEcsWithGithubAction.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can become a challenge for architecture and platform infrastructure teams due to the integration complexity with many platforms: Source Control, Continuous Integration/Continuous Delivery (CI/CD), Artifact Repositories, Quality, SAST/SCA, and Deployment. Many DevOps leaders believe that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.
```
Comment (yhippa) [inline posts/deployToEcsWithGithubAction.md]: Please spell out initialisms prior to using them. Also, decapitalize nouns unless they are proper nouns.

Comment (amineoualialami) [reply posts/deployToEcsWithGithubAction.md]: I changed SAST/SCA by just security.
I don't want here to spell out: Static Application Security Testing (SAST), Dynamic Application Security Testing (DAST), Security Control Assessment (SCA), Seems too much for me.

**Pull request merged**